src/Redux/Features/categoriesSlice.js

```diff
@@ -0,0 +1,23 @@
+// src/features/categoriesSlice.js
+
+import { createSlice } from '@reduxjs/toolkit';
```

Comment:
- [ ] Dear @Errytagedesign, Good job addressing the changes requested by the previous reviewer, but while doing so I notice you have commented out some old codes.  Please consider removing `commented codes` as it will make your codebase messy, Remember if you ever like to see the codes you have written once you can use your commit history.

Reply:
Thank you for pointing this out, however, is it important to request for changes just to asked me to remove commented codes?

Reply:
Dear @Errytagedesign, Its a required change according to the microverse guideline for a project that has a commented code. I'm following the guideline and it is to make sure you have taken such issues seriously.